concepts/0167-consent-lifecycle/README.md

```diff
@@ -0,0 +1,668 @@
+# Aries RFC 0167: Consent Lifecycle
+
+- Authors: \
+    Jan Lindquist, Dativa\
+    Mark Lizar, OpenConsent\
+    Harshvardhan J. Pandit, ADAPT Centre, Trinity College Dublin
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2019-08-07
+- Status Note: This RFC is based on [[old
```

Comment:
Hi Jan, the purpose of the "Supersedes" field is to include the link in that field so the Status Note doesn't have to reflect that.  Could you please update to move the link/text down to that field and reword the status note accordingly?